style/svgs/color_palette.svg

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="100px" height="100px" viewBox="0 0 100 100" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
we could ralso run `svgo` to optimize svgs - see `yarn optimize:svg` in `editor-mvp` (react-client and czi-prosemirror)